### PR TITLE
CLXP-63 Learning priority card

### PR DESCRIPTION
**Pull request by adamska27**
**Detailed purpose of the PR**

- update existing Tag component
- create a new circularProgressBar component
- create a new LearningPriorityCard component

https://go1web.atlassian.net/jira/software/projects/CLXP/boards/626?selectedIssue=CLXP-63

**Result and observation**

image

chromatic link: https://6622682e70ecd8ecf5027fa1-lpswutqqwp.chromatic.com/?path=/story/molecule-learningprioritycard--certification-type-100

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by HuyIT9898** [comment]
Can you please add a chromatic review link for this?

**Comment by adamska27** [comment]
> https://6622682e70ecd8ecf5027fa1-lpswutqqwp.chromatic.com/?path=/story/molecule-learningprioritycard--certification-type-100

done 👍 

https://6622682e70ecd8ecf5027fa1-lpswutqqwp.chromatic.com/?path=/story/molecule-learningprioritycard--certification-type-100

File: packages/@coorpacademy-components/src/molecule/learning-priority-card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  const buttonExploreProps = {
+    customStyle: {
+      backgroundColor: hovered ? primarySkinColor : convert(`color(${primarySkinColor} a(0.07))`),
+      color: hovered ? '#FFFFFF' : primarySkinColor,
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onClick,
+    'aria-label': `, ${exploreLocale}`,
+    label: exploreLocale,
+    'data-name': 'learner-skill-card-explore-button',
```
[the remark this answers is not in the document]
Comment (adamska27) [reply packages/@coorpacademy-components/src/molecule/learning-priority-card/index.tsx:37]: ah yes, I missed that, thanks!

File: packages/@coorpacademy-components/src/molecule/learning-priority-card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+const LearningPriorityCard = (props: LearningPriorityCardPropTypes, context: WebContextValues) => {
+  const {
+    tag: {label: tagLabel, type},
+    title,
+    progression,
+    onClick
+  } = props;
+  const translate = GetTranslateFromContext(context);
+  const skin = GetSkinFromContext(context);
+  const exploreLocale = translate('Explore');
```
[the remark this answers is not in the document]
Comment (adamska27) [reply packages/@coorpacademy-components/src/molecule/learning-priority-card/index.tsx:21]: it should be ? @wernerdurand

File: packages/@coorpacademy-components/src/molecule/learning-priority-card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import {WebContextValues} from '../../atom/provider/web-context';
+import Tag from '../../atom/tag';
+import ButtonLink from '../../atom/button-link';
+import CircularProgressBar from '../../atom/circular-progress-bar';
+import {LearningPriorityCardPropTypes, propTypes} from './types';
+import style from './style.css';
+
+const LearningPriorityCard = (props: LearningPriorityCardPropTypes, context: WebContextValues) => {
+  const {
+    tag: {label: tagLabel, type},
```
[the remark this answers is not in the document]
Comment (adamska27) [reply packages/@coorpacademy-components/src/molecule/learning-priority-card/index.tsx]: yes indeed, we can remove the useless tag type props

**Review by HuyIT9898 — APPROVED** [review]

**Pull request merged**